aim/db/migration/alembic_migrations/versions/fccf2c4f6282_erspan.py

```diff
@@ -0,0 +1,220 @@
+# Copyright (c) 2020 Cisco Systems
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+"""Tables for erspan
+Revision ID: fccf2c4f6282
+Revises: 794ad00f3080
+Create date: 2020-08-17 13:05:03.236000000
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'fccf2c4f6282'
+down_revision = '794ad00f3080'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.create_table(
+        'aim_span_vsource_grp',
+        sa.Column('aim_id', sa.String(64), nullable=False),
+        sa.Column('name', sa.String(64), nullable=False),
+        sa.Column('display_name', sa.String(256), nullable=False, default=''),
+        sa.Column('admin_st', sa.Enum('start', 'stop')),
+        sa.Column('monitored', sa.Boolean, nullable=False, default=False),
+        sa.Column('epoch', sa.BigInteger(), nullable=False,
+                  server_default='0'),
+        sa.PrimaryKeyConstraint('aim_id'),
+        sa.UniqueConstraint('name',
+                            name='uniq_aim_span_vsource_grp_identity'),
+        sa.Index('idx_aim_span_vsource_grp_identity', 'name'))
+
+    op.create_table(
+        'aim_span_vsource',
+        sa.Column('aim_id', sa.String(64), nullable=False),
+        sa.Column('name', sa.String(64), nullable=False),
+        sa.Column('vsg_name', sa.String(64), nullable=False),
+        sa.Column('display_name', sa.String(256), nullable=False, default=''),
+        sa.Column('dir', sa.Enum('in', 'out', 'both')),
+        sa.Column('monitored', sa.Boolean, nullable=False, default=False),
+        sa.Column('epoch', sa.BigInteger(), nullable=False,
+                  server_default='0'),
+        sa.PrimaryKeyConstraint('aim_id'),
+        sa.UniqueConstraint('vsg_name', 'name',
+                            name='uniq_aim_span_vsource_identity'),
+        sa.Index('idx_aim_span_vsource_identity', 'vsg_name', 'name'),
+        sa.ForeignKeyConstraint(
```

Comment:
Please cross-check the use of ForeignKeyConstraints in the DB migration with those of the model classes. I think the model classes are correct (i.e. uses aim IDs for foreign keys).